### ISSUE TITLE
If you do a game with included regions of advanced and job rando none 2 of the checks ask for more jobs than exist in an advanced game

### ISSUE THREAD

**Issue by Emerassi** [Emerassi/CrystalProjectAPWorld#144]
[root at 2025-06-07 16:53:06,669]: Could not access required locations for accessibility check. Missing: [Capital Sequoia NPC - Luxury Store Lazy Guard 2 (a), Sara Sara Bazaar NPC - Ferry Ticket Agent grants Ferry Pass in case you hate children (a)]

Exclude the 2 locations if you're on advanced and job rando none.

**Comment by Emerassi** [comment]
Bumped the requirements down to 11 and renamed the locations to call out how jobs you need.  Also updated the location name for the 6 job first requirement on the luxury store.
